docs/problems/nn.rst

```diff
@@ -10,6 +10,7 @@ Dataset
 .. autosummary::
     :toctree: _autosummary
 
-    dataset.create_gaussian_mixture_samplers
+    dataset.gaussian_mixture_samplers
+    dataset.uniform_mixture_samplers
     dataset.Dataset
     dataset.GaussianMixture
```

Comment:
I think this can be removed from here, users should interact with the `{gaussian,uniform}_mixture_samplers` only.